src/shared/styles/react-select.global.css

```diff
@@ -19,6 +19,11 @@
   line-height: 50px;
 }
 
+.Select--single .Select-value {
```

Comment:
It shouldn't have, since that one's a multi-select, that should use the class `.Select--multi`. I basically had to add that rule here so that single-select React-select components get the same styling.

I'll double check before merging tho.

Reply:
Just checked, it's good to go in the `/code-schools` page. The new CSS rule doesn't target it.